enclave/src/rpc/mocks/rpc_gateway_mock.rs

```diff
@@ -138,9 +139,9 @@ impl RpcGateway for RpcGatewayMock {
         _main_account: AccountId,
         _proxy_acc: Option<AccountId>,
         _order: Order,
-    ) -> Result<(), GatewayError> {
+    ) -> Result<RequestId, GatewayError> {
         match &self.order_uuid {
-            Some(_) => Ok(()),
+            Some(_) => Ok(0u128),
```

Comment:
Could you change this to 0 as RequestId (or something alike)

Reply:
done